pkg/orchestrator/scanestimationwatcher/watcher.go

```diff
@@ -169,15 +170,59 @@ func (w *Watcher) Reconcile(ctx context.Context, event ScanEstimationReconcileEv
 			return err
 		}
 	case models.ScanEstimationStateStateDone, models.ScanEstimationStateStateFailed:
-		logger.Debug("Reconciling ScanEstimation is skipped as it is already finished.")
-		fallthrough
+		if err = w.reconcileDone(ctx, scanEstimation); err != nil {
+			return err
+		}
 	default:
 		return nil
 	}
 
 	return nil
 }
 
+func (w *Watcher) reconcileDone(ctx context.Context, scanEstimation *models.ScanEstimation) error {
+	if scanEstimation.EndTime == nil {
+		return nil
+	}
+	if scanEstimation.TTLSecondsAfterFinished == nil {
+		return nil
+	}
```

Comment:
same as AssetScanEstimations we should reconcile defaults for these before this point, and if they are not set it should be an error